plugins/BEdita/Core/src/Shell/BeditaShell.php

```diff
@@ -349,7 +351,13 @@ protected function saveConnectionData()
      */
     protected function dbConnectionUserInput()
     {
-        $this->userInputData = [];
+        $this->userInputData = ['host' => 'localhost', 'port' => '', 'database' => '', 'username' => '', 'password' => ''];
+        $this->userInputData['driver'] = $this->in('Driver?', ['Mysql', 'Postgres', 'Sqlite'], 'Mysql');
+        if ($this->userInputData['driver'] === 'Sqlite') {
+            $this->userInputData['database'] = $this->in('Sqlite database file path?', null, TMP . 'be4.sqlite');
+
+            return;
+        }
         $this->userInputData['host'] = $this->in('Host?', null, 'localhost');
         $this->userInputData['port'] = $this->in('Port?', null, '3306');
```

Comment:
Default port suggested should not be always 3306 (mysql default port). I.e. postgres default port is 5432.